src/components/AppComponents/Wall/Wall.tsx

```diff
@@ -0,0 +1,25 @@
+import {
+  OakBox,
+  OakThemeProvider,
+  oakDefaultTheme,
+} from "@oaknational/oak-components";
+import { PropsWithChildren } from "react";
+
+/**
+ * Hides content both visually and from AT
+ *
+ * Intended to enable client-side authorisation while still allowing the content
+ * to be spidered
+ */
+export const Wall = ({ children }: PropsWithChildren) => {
+  return (
+    <OakThemeProvider theme={oakDefaultTheme}>
+      <OakBox as="noscript" $position="absolute">
+        JavaScript is required to access this content
+      </OakBox>
+      <OakBox data-testid="content" aria-hidden="true" $opacity="transparent">
+        {children}
+      </OakBox>
+    </OakThemeProvider>
+  );
+};
```

Comment:
@mantagen raised a point that we would still want restricted pages to be accessible to spiders for SEO purposes. So doing the auth client-side actually makes some sense now. We'll be requiring authorisation on the downloads API in an upcoming ticket so downloads will still be properly restricted.

This component hides the page for the split second while Clerk loads and reads the user session before it can redirect to login or reveal the page content